make

```diff
@@ -42,44 +79,30 @@ runtests() {
     ../build/mi test mexpr
     cd ..
     export MCORE_STDLIB='@@@'
-    build/mi test stdlib)
-}
-
-# Run the test suite including external functions tests
-runtests_ext() {
-    runtests
-    (cd test
-     ../build/mi test ext/*)
-    build/mi test stdlib/ext/*
+    build/mi test stdlib
+    if [[ -n $MI_ENABLE_PYTHON ]]; then
+        runtests_py
+    fi
+    if [[ -n $MI_ENABLE_SUNDIALS ]]; then
+        runtests_ext
+    fi
+    )
 }
 
 case $1 in
-    # with external functions integration
-    externals-test)
```

Comment:
Mentioning this for completeness since it was mentioned during the meeting, it'd be good to add a `make test-all` or similar that includes all supported externals and runs all tests.

Reply:
This is added now.